lib/kitchen/configurable.rb

```diff
@@ -390,8 +390,8 @@ def plugin_version(version) # rubocop:disable Style/TrivialAccessors
       def diagnose
         {
           class: name,
-          version: @plugin_version,
-          api_version: @api_version,
+          version: @plugin_version ||= nil,
```

Comment:
Addendums:

+ `@plugin_version || nil` won't prevent the uninitialized variable warning. `||=` does.
+ In all circumstances, the uninitialized instance variables were being used as though it was expected to be initialized and be falsey to trigger alternate behavior.